functests/1_performance/performance.go

```diff
@@ -340,8 +342,161 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("cannot find MachineConfig object %s", initialKey.Name))
 		})
 	})
+
+	Context("Verify API Conversions", func() {
+		It("[test_id:35887] Verifies v1 <-> v1alpha1 conversions", func() {
+
+			By("Checking v1 -> v1alpha1 conversion")
+			v1alpha1Profile := &performancev1alpha1.PerformanceProfile{}
+			key := types.NamespacedName{
+				Name:      profile.Name,
+				Namespace: profile.Namespace,
+			}
+
+			err := testclient.Client.Get(context.TODO(), key, v1alpha1Profile)
+			Expect(err).ToNot(HaveOccurred(), "Failed getting v1alpha1Profile")
+			Expect(verifyV1alpha1Conversion(v1alpha1Profile, profile)).ToNot(HaveOccurred())
+
+			By("Checking v1alpha1 -> v1 conversion")
+			v1alpha1Profile.Name = "v1alpha"
+			v1alpha1Profile.ResourceVersion = ""
+			Expect(testclient.Client.Create(context.TODO(), v1alpha1Profile)).ToNot(HaveOccurred())
+			defer func() { Expect(testclient.Client.Delete(context.TODO(), v1alpha1Profile)).ToNot(HaveOccurred()) }()
+
+			v1Profile := &performancev1.PerformanceProfile{}
+			key = types.NamespacedName{
+				Name:      v1alpha1Profile.Name,
+				Namespace: v1alpha1Profile.Namespace,
+			}
+
+			err = testclient.GetWithRetry(context.TODO(), key, v1Profile)
```

Comment:
why do we need WithRetry again?

Reply:
we created a profile, it may take some time to be created